### ISSUE TITLE
Usenet target?

### ISSUE THREAD

**Issue by SanderM2** [rclone/rclone#1488]
Since usenet providers offer a retention of 3000 days (giganews only?) it is already used by some people for personal backups by encrypting their backups and uploading it to usenet.

I'm not sure how much work it'll be to add support for this to rclone but I would like to know if you would consider implementing it? It probably needs a little different approach compared to the something like dropbox but in theory it should be possible.

**Comment by scriptzteam** [comment]
Yup there are some go nntp samples -> https://github.com/dustin/go-nntp & https://github.com/chrisfarms/nntp and more ...

P.S.: I created list of popular groups auto updating in my own nntp server (storing about 9000TB of data, ~850 days retention) :D https://github.com/scriptzteam/UseNet-Newsgroups-Stats xD

**Comment by ncw** [comment]
The idea of backing up to nntp is both bonkers and brilliant!

**Comment by SanderM2** [comment]
Does this mean you're thinking about implementing it?

**Comment by ncw** [comment]
@SanderM2 I have too much on my plate at the moment, but if someone wanted to have a go I'll provide help.

**Comment by kaileu** [comment]
Maybe this is a start:
https://github.com/nzbfs/go-nzbfs

**Comment by coou** [comment]
Interesting concept! if anyone is looking for rclone 
sync to nntp - they can probably use webdav remote together with this: https://github.com/retroplasma/binsync
more info here: 
https://gist.github.com/retroplasma/264d9fed2350feb19f977575981bb914

according to the author - binsync was just a concept; haven't tested it myself.
